openmdao/utils/variable_table.py

```diff
@@ -308,3 +308,124 @@ def _write_variable(out_stream, row, column_names, var_dict, print_arrays):
             indented_lines = [(left_column_width + indent_inc) * ' ' +
                               s for s in out_str.splitlines()]
             out_stream.write('\n'.join(indented_lines) + '\n')
+
+
+def write_options(pathname, opt_list, out_stream=_DEFAULT_OUT_STREAM):
```

Comment:
a nit, but I would name this `write_options_table` to be consistent with the other functions in this file.